infra/helm/meshdb/values.yaml

```diff
@@ -83,7 +91,13 @@ redis:
   port: 6379
   podSecurityContext: {}
   securityContext: {}
-  resources: {}
+  resources:
+    limits:
+      cpu: 1
+      memory: 512Mi
+    requests:
+      cpu: 50m
+      memory: 30Mi
```

Comment:
Redis _really_ doesn't like not being pinned, especially as an in-memory cache. I'd say pin it to 256Mi and like..... 250mcore. If that doesn't work, bump it.